modules/nw-metallb-addresspool-cr.adoc

```diff
@@ -49,3 +49,31 @@ Specify `false` if you want explicitly request an IP address from this pool with
 The default value is `true`.
 
 |===
+
+You can assign IP addresses from an `IPAddressPool` to services and namespaces by configuring the `spec.serviceAllocation` specification.
+
+.MetalLB IPAddressPool custom resource `spec.serviceAllocation` subfields
+[cols="1,1,3a", options="header"]
+|===
+
+|Field
+|Type
```

Comment:
type is incorrect for all the fields, refer to https://github.com/metallb/metallb/blob/main/website/content/apis/_index.md#serviceallocation
you could also pull description from there to make it more relevant.